app/views/users/_users_form.html.erb

```diff
@@ -0,0 +1,65 @@
+<% if user.errors.any? %>
+  <div id="error_explanation">
+    <h2>
+      <%= pluralize(user.errors.count, "error") %>
+      prohibited this user from being saved:
+    </h2>
+
+    <ul>
+    <% user.errors.full_messages.each do |msg| %>
+      <li><%= msg %></li>
+    <% end %>
+    </ul>
+  </div>
+<% end %>
+
+<!--
+<form action="/users" method="post">
```

Comment:
Rails can generate the route for you by using `users_path`. Furthermore, you can take a look at your routes by executing the following command in your terminal: `$ rails routes`. Take a look at all user route paths and the action they are mapped to (to have rails generate the path for you, just use the specified prefix and append `_path` to it. It's always good to check for how your routes are structured, either when debugging or just for sanity check.

[`OPTIONAL`]